application/src/main/java/org/thingsboard/server/service/security/auth/oauth2/Oauth2AuthenticationSuccessHandler.java

```diff
@@ -99,6 +101,8 @@ public void onAuthenticationSuccess(HttpServletRequest request,
             clearAuthenticationAttributes(request, response);
             getRedirectStrategy().sendRedirect(request, response, baseUrl + "/?accessToken=" + accessToken.getToken() + "&refreshToken=" + refreshToken.getToken());
         } catch (Exception e) {
+            log.error("Error occurred during processing authentication success result. " +
```

Comment:
Let's make it debug. This is sensitive info that we can't log to the file by default.  You can always change the log level for the class in logback.xml without restart. Also, you can add a template to logback.xml to change the log level with appropriate comment.